File: springTest/src/main/java/dawn/springTest/domain/Todo.java
package dawn.springTest.domain;

import jakarta.persistence.*;
import lombok.Data;
import lombok.RequiredArgsConstructor;

@RequiredArgsConstructor
@Entity
@Data
@Table(name = "TODOLIST")
public class Todo {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String content;
    private String limitDateTime;
    private String tag;
    private String ownerEmail;
    private String finish;

    public Todo(String content, String limitDateTime, String tag, String ownerEmail,String finish){
        this.content = content;
        this.limitDateTime = limitDateTime;
        this.tag = tag;
        this.ownerEmail = ownerEmail;
        this.finish = finish;
    }
}

File: app/src/main/java/com/dawn/springtest/model/Todo.kt
package com.dawn.springtest.model

import org.intellij.lang.annotations.Identifier
import java.time.LocalDateTime

data class Todo(
    val id: String="",
    val content: String = "",
    val limitDateTime: String = "",
    val tag: String = "",
    val ownerEmail: String = "",
    val finish: String=""
)
